omnigibson/prims/rigid_prim.py

```diff
@@ -89,6 +89,14 @@ def _post_load(self):
         # run super first
         super()._post_load()
 
+        # Apply rigid body and mass APIs
+        self._rigid_api = UsdPhysics.RigidBodyAPI(self._prim) if self._prim.HasAPI(UsdPhysics.RigidBodyAPI) else \
+            UsdPhysics.RigidBodyAPI.Apply(self._prim)
+        self._physx_rigid_api = PhysxSchema.PhysxRigidBodyAPI(self._prim) if \
+            self._prim.HasAPI(PhysxSchema.PhysxRigidBodyAPI) else PhysxSchema.PhysxRigidBodyAPI.Apply(self._prim)
+        self._mass_api = UsdPhysics.MassAPI(self._prim) if self._prim.HasAPI(UsdPhysics.MassAPI) else \
+            UsdPhysics.MassAPI.Apply(self._prim)
+
```

Comment:
This is needed to get the PrimitiveObject loading work (c.f. @ChengshuLi )

Reply:
RigidPrimView already supposedly adds these so I don't know why this is happening, but if we need to keep it please go ahead and convert it into the below format:

```
if not self._prim_HasAPI(...):
   ...API.Apply(self._prim)
```

etc. (e.g. don't actually save those APIs as fields. I don't think we need them)